src/script/engine.zig

```diff
@@ -265,6 +267,24 @@ pub const Engine = struct {
         return error.EarlyReturn;
     }
 
+    /// OP_TOALTSTACK: Puts the value onto the top of the alt stack, and removes it from the main stack.
+    ///
+    /// # Returns
+    /// - `EngineError`: If an error occurs during execution
+    fn opToAltStack(self: *Engine) EngineError!void {
+        const value = try self.stack.pop();
+        try self.alt_stack.pushElement(value);
+    }
+    
+    /// OP_FROMALTSTACK: Puts the value onto the top of the main stack, and removes it from the alt stack.
+    ///
+    /// # Returns
+    /// - `EngineError`: If an error occurs during execution
```

Comment:
So now this section of the comment is useless, you can remove it

Reply:
got it